keyboards/hotdox/hotdox.c

```diff
@@ -0,0 +1,59 @@
+#include "hotdox.h"
+#include "backlight.h"
+#include "quantum.h"
+
+extern inline void ergodox_right_led_1_on(void);
+extern inline void ergodox_right_led_2_on(void);
+extern inline void ergodox_right_led_3_on(void);
+extern inline void ergodox_right_led_on(uint8_t led);
+
+extern inline void ergodox_right_led_1_off(void);
+extern inline void ergodox_right_led_2_off(void);
+extern inline void ergodox_right_led_3_off(void);
+extern inline void ergodox_right_led_off(uint8_t led);
+
+extern inline void ergodox_led_all_off(void);
+
```

Comment:
It looks like you may want an `ergodox_board_led_on` and `ergodox_board_led_off` function here.  Adding this should fix all of the travis errors. 

https://github.com/qmk/qmk_firmware/blob/master/keyboards/ergodox_ez/ergodox_ez.c#L3-L23